src/particle/lhcb/__init__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018-2021, Eduardo Rodrigues and Henry Schreiner.
+#
+# Distributed under the 3-clause BSD license, see accompanying file LICENSE
+# or https://github.com/scikit-hep/particle for details.
+
+from __future__ import absolute_import
+
+from typing import Tuple
+
+from .utils import from_lhcb_name, to_lhcb_name
+
+__all__ = (
```

Comment:
Given that this is an experiment specific submodule I think it is fine to add `LHCbName2PDGIDBiMap` to the list of "import *".